### PR TITLE
Add version number and check that it always matches in CI

### LINKED ISSUE #168: Add version number to schema and generated floorplans

### PR DESCRIPTION
**Pull request by bgschiller**
fix #168

### CODE REVIEW

File: schema/geometry_schema.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
+	"version": "0.3.0",
```
Comment (bgschiller) [inline schema/geometry_schema.json:3]: @macumber is this acceptable in the schema? I've never used json-schema before this.

Comment (macumber) [reply schema/geometry_schema.json:3]: Let me add it to the schema, I have a fancy editor for handling json schema.  You are looking to add a version key at the top level, right above application and project?

Comment (macumber) [reply schema/geometry_schema.json:3]: Here is a pull request that includes a version property https://github.com/NREL/openstudio-geometry-editor/pull/175

File: .travis.yml [hunk 2]
```diff
@@ -3,3 +3,4 @@ node_js:
   - "6"
 notifications:
   email: false
+script: npm run test && node check_version.js
```
Comment (gh-23378) [inline .travis.yml:6]: good idea adding this into CI, it seems like something that would be easy to forget to update

**Review by gh-23378 — APPROVED** [review]

File: schema/geometry_schema.json (hunk 1 above)
Comment (bgschiller) [reply schema/geometry_schema.json:3]: @macumber Ah, that's a useful addition as well. But I was thinking we would want a version *on the schema*. It looks like your PR adds lines that say "a valid floorplan adhering to this schema will have a version". I'd like to also say "you're looking at version 0.3.0 of the schema".

Happy to roll this part back though if you think it doesn't make sense.

**Pull request merged**